learning_assistant/views.py

```diff
@@ -103,6 +136,12 @@ def post(self, request, course_run_id):
         )
         status_code, message = get_chat_response(prompt_template, message_list)
 
+        self.__save_user_interaction(
```

Comment:
This needs to be gated by a waffle flag, which also needs to be created.

Reply:
Oh shoot. I totally forgot about it.

Reply:
Added waffle flag and gated the functionality behind it.